### DISCUSSION TITLE
Error: Forbidden Your client does not have permission to get URL /about from this server.

### DISCUSSION THREAD

**Discussion by sergiuchuckmisha** [vercel/next.js#15516] [Help]
I'm trying to run [with-firebase-hosting](https://github.com/vercel/next.js/tree/canary/examples/with-firebase-hosting) demo

When I call function url 
https://us-central1-project-id.cloudfunctions.net/nextjsFunc
it works

but when I click on [About](https://us-central1-<project-id>.cloudfunctions.net/about)
button it throws 

"Error: Forbidden
Your client does not have permission to get URL /about from this server."

error

looks like smth is wrong with code, 

because link in button should point not to 
https://us-central1-project-id.cloudfunctions.net/about
but to 
https://us-central1-project-id.cloudfunctions.net/nextjsFunc/about

**Comment by francisrod01** [comment]
> Note: As of January 15, 2020, HTTP functions require authentication by default. You can specify whether a function allows unauthenticated invocation at or after deployment.

https://cloud.google.com/functions/docs/securing/managing-access-iam#allowing_unauthenticated_function_invocation
